src/System.Net.Http/src/uap/System/Net/HttpClientHandler.cs

```diff
@@ -701,6 +657,15 @@ private void SetOperationStarted()
                     _rtFilter.IgnorableServerCertificateErrors.Add(RTChainValidationResult.Untrusted);
                     _rtFilter.IgnorableServerCertificateErrors.Add(RTChainValidationResult.WrongUsage);
                     _rtFilter.ServerCustomValidationRequested += RTServerCertificateCallback;
+
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.Expired);
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.IncompleteChain);
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.InvalidName);
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.RevocationFailure);
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.RevocationInformationMissing);
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.Untrusted);
+                    _rtFilterWithNoCredentials.IgnorableServerCertificateErrors.Add(RTChainValidationResult.WrongUsage);
+                    _rtFilterWithNoCredentials.ServerCustomValidationRequested += RTServerCertificateCallback;
```

Comment:
In general, that callback can be called several times even without redirection involved.  For example, start multiple parallel requests to the same or different servers.  There is only one callback defined per filter.  So, it is already something that a developer needs to handle (i.e. handle multiple calls on the callback.